Makefile

```diff
@@ -43,7 +43,7 @@ check_flake8:
 	flake8 $(python-packages) ide/gnatstudio
 
 check_pylint:
-	pylint $(python-packages)
+	PYTHONPATH=. pylint $(python-packages)
```

Comment:
The need for having `PYTHONPATH` set most certainly prevents the use of pylint in some IDEs. Is there another solution?

Reply:
I found no way to do that, initially. But as we all know there is no hack [you cannot find on Stackoverflow](https://stackoverflow.com/a/39207275/8058522) ;)  See c5c7f148.